hls4ml/utils/config.py

```diff
@@ -281,8 +188,9 @@ def make_layer_config(layer):
     model_config['Precision'] = default_precision
     model_config['ReuseFactor'] = default_reuse_factor
     model_config['Strategy'] = 'Latency'
+    model_config['BramFactor'] = 1_000_000_000
     #model_config['Compression'] = False
-    #model_config['Trace'] = False
+    model_config['TraceOutput'] = False
```

Comment:
Does this key mean all layers are traced if this is set to `True`?

Reply:
No, this key simply turns on tracing. Then all layers that have `trace` set to true will be traced.